components/backdrop/backdrop.js

```diff
@@ -14,6 +14,7 @@ class Backdrop extends LitElement {
 		return {
 			shown: { type: Boolean },
 			forTarget: { type: String, attribute: 'for-target' },
+			cutOut: { type: Boolean, attribute: 'cut-out' },
```

Comment:
I think something like `transitions: 'show-hide | show-only'` (string) or `no-animate-hide` (boolean) would be clearer.   In either case, if we ever need to expand on this to include _show_, we can add another valid string value `show-only`, or with the boolean approach another property, `no-animate-show`.  The `no-*` naming has some precedent in our other components.

Reply:
Sounds good, I'll change it to `no-animate-hide`. I know my name was terrible ^^
And yes, design explicitly wanted the animation to be a lot slower. I think it's about giving feedback that something is happening, while hacking user perception when the save process is being slow. 1.2s was arrived at by demoing different values on my local instance.

Reply:
One thing to note (not sure if the designer is aware) is that if the user has reduced motion preference set, the backdrop will be shown immediately.

Reply:
Right, yes. I had mentioned this before in passing, but will make sure they are aware.